### PR TITLE
Port boot_delay to /bin/sh. Minor Makefile/configure cleanups

### PR DESCRIPTION
**Pull request by nabijaczleweli**
```
$ hyperfine --prepare './boot_delay.py 0' -N './boot_delay.py 0' './boot_delay.sh 0'
Benchmark 1: ./boot_delay.py 0
  Time (mean ± σ):      21.8 ms ±   2.5 ms    [User: 16.6 ms, System: 5.1 ms]
  Range (min … max):    17.7 ms …  25.8 ms    61 runs

Benchmark 2: ./boot_delay.sh 0
  Time (mean ± σ):       1.3 ms ±   0.2 ms    [User: 0.9 ms, System: 0.3 ms]
  Range (min … max):     1.0 ms …   1.8 ms    121 runs

Summary
  './boot_delay.sh 0' ran
   16.18 ± 2.63 times faster than './boot_delay.py 0'
```

Check shell programs with shellcheck.

Read VERSION directly. Make builddirs in one go. Use full `$(outputs): $(builddirs)` instead of order-only and then drop `all: $(builddirs)`.

In configure, use function for schedules like in set_enable_flag.

### CODE REVIEW

**Comment by a-detiste** [comment]
I guess the C rewrite in `systemd-cron-c` rewrite is even faster.
This other project has been on hold from 2016 to 2023 but I still hope to finish some day.

**Comment by a-detiste** [comment]
And the maybe with a more recent `systemd` there would be a new syntax to reach the same goal, which is having Persistent timers not firing too early after/(during !) boot.

File: configure [hunk 1]
```diff
@@ -179,55 +179,28 @@ do
...
-    schedules_not="${schedules_not} boot"
-fi
+check_sched_enabled()
+{
+    if eval [ "\"\${enable_${1}}\"" = \'yes\' ]; then
+        schedules="${schedules} ${2}"
+    else
+        schedules_not="${schedules_not} ${2}"
+    fi
+}
```
Comment (a-detiste) [inline configure:190]: thank you for this, you can also add yourself to the CHANGELOG. ("configure & Makefile cleanup ...")
I will publish a new release on 15/8

**Comment by a-detiste** [comment]
> And the maybe with a more recent `systemd` there would be a new syntax to reach the same goal, which is having Persistent timers not firing too early after/(during !) boot.

https://github.com/systemd/systemd/issues/26943#issuecomment-1481360560

**Comment by nabijaczleweli** [comment]
As for the shell port, I think it makes little sense to reimplement in C since you get identical performance from the shell, while not paying for a whole python interpreter to just parse an integer makes sense since this will be running in jobs. OTOH, it may make sense to straight-up inline it into the services since it's *so* trivial. Let's see.

**Pull request merged**